### ISSUE TITLE
Not a desktop App

### ISSUE THREAD

**Issue by francogrex** [oOthkOo/whatsapp-desktop#53]
This is just a copy of the whatsapp web. I expected this to be a fully functional whatsapp app like the one on IOS/android phone or windows (where you can make phone calls). But it's not it.

**Comment by lifeboy** [comment]
Indeed, the name of the project should be changed to reflect this.

**Comment by OmegaAOL** [comment]
So you people didn't read the description where it clearly states that this is an ELECTRON app? smh

**Comment by lifeboy** [comment]
> So you people didn't read the description where it clearly states that this is an ELECTRON app? smh

No-one, but maybe a developer, even knows what is mean to call an app an electron app...

**Comment by OmegaAOL** [comment]
> > So you people didn't read the description where it clearly states that this is an ELECTRON app? smh
> 
> No-one, but maybe a developer, even knows what is mean to call an app an electron app...

There is already an official Whatsapp desktop program much better than this one, and it's been out for over 3 years at this point. I feel like someone who looks for an obtuse github repo of a fake desktop app last updated 3 years ago, ignores that there is an official app and chooses to download this ancient electron app instead should be well aware of the difference. 

If you don't know, just download the official app - it isn't that hard, and is a lot easier than using this
